docs/templates/layout.html

```diff
@@ -1,6 +1,8 @@
 {% extends "!layout.html" %}
 
 {%- block extrahead %}
+    <!-- Switch the script tag for the docs to work locally. -->
+    <!-- <script src="/_static/candela.js"></script> -->
```

Comment:
There must be a way to do this using templating magic? We can investigate in a separate PR.